pkg/harvester/components/settings/storage-network.vue

```diff
@@ -84,9 +84,13 @@ export default {
       const clusterNetworks = this.$store.getters[`${ inStore }/all`](HCI.CLUSTER_NETWORK) || [];
 
       return clusterNetworks.map((n) => {
+        const readyCondition = (n?.status?.conditions || []).find(c => c.type === 'ready') || {};
```

Comment:
I think we can add isReady in the cluster-network model to determine whether the CR can be used, so that the attribute can be reused elsewhere。

Reply:
Done